### ISSUE TITLE
Setup with existing Nginx Proxy Manager

### ISSUE THREAD

**Issue by m3tc0n** [fractalnetworksco/selfhosted-gateway#57]
Hi, I have an existing NPM container running on my home server. Is it possible to point the traffic from the VPS to the NPM?

For example: VPS (npm.domain.com) > tunnel > NPM (immich.domain.com > 192.168.30.3:2381) > Immich container with local IP 192.168.30.3:2381.

**Comment by thebalaa** [comment]
this should work no problem, would you like to route all domains from the VPS to the NPM?

**Comment by m3tc0n** [comment]
Yes, I have about 5 services I want to expose through the vps with 5 different subdomains. What is the best thing to do? Is it possible to only setup a tunnel gateway link to NPM on my home server and route all domains through it?

**Comment by paul-eff** [comment]
I'd also be interested on how to accomplish this!
